### PR TITLE
fix: bug 2356

### PR DESCRIPTION
**Pull request by LarsLiden**
update train dialog validity flags on replay from train dialog list view

### CODE REVIEW

File: src/routes/Apps/App/TrainDialogs.tsx [hunk 1]
```diff
@@ -1071,6 +1071,21 @@ class TrainDialogs extends React.Component<Props, ComponentState> {
                 this.props.trainDialogReplayThunkAsync as any,
             )
 
+            // Get new activities to check for errors or warnings
+            const teachWithActivities = ((await this.props.fetchActivitiesThunkAsync(this.props.app.appId, newTrainDialog, this.props.user.name, this.props.user.id) as any) as Promise<CLM.TeachWithActivities>)
+            const replayError = DialogUtils.getMostSevereReplayError((await teachWithActivities).activities)
```
Comment (mattmazzola) [inline src/routes/Apps/App/TrainDialogs.tsx]: It looks like you have incorrect `await` statement due to wrong types above.  Either way two `await`s is weird and wouldn't recommend this syntax in inline `await`s

Comment (LarsLiden) [reply src/routes/Apps/App/TrainDialogs.tsx]: Ugh.. Sloppy coding. thx

Comment (LarsLiden) [reply src/routes/Apps/App/TrainDialogs.tsx]: Wow. It's vscode that stuck that in.  I typed "a", TAB for auto-complete, and....

vccode

**Pull request merged**